src/interfaces/ITextMessage.ts

```diff
@@ -4,7 +4,8 @@ export interface ITextMessage {
   author: string;
   roomID: string;
   uniqueID: string;
-  message?: string;
+  message: string | ISocketFile;
```

Comment:
Message må ikke nødvendigvis være med. Man må kunne sende en mld med bare string.

ISocketFile blir ikke brukt lenger, så den trengs ikke

```suggestion
  message?: string;
```

Reply:
Er dette fikset?